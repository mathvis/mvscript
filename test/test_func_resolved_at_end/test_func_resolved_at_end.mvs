[fwd] func a();
[fwd] func b();
a();
b();
func a() {}
func b() {}
