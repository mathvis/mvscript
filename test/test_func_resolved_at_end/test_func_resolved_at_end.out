Decl (FunctionDef (FunctionIdentifier "a") [] void Nothing)
Decl (FunctionDef (FunctionIdentifier "b") [] void Nothing)
Expr (FunctionCall (FunctionIdentifier "a") [])
Expr (FunctionCall (FunctionIdentifier "b") [])
Decl (FunctionDef (FunctionIdentifier "a") [] void (Just (Block (FunctionBlock void) [])))
Decl (FunctionDef (FunctionIdentifier "b") [] void (Just (Block (FunctionBlock void) [])))
